docs/topics/api/addons.rst

```diff
@@ -60,15 +60,17 @@ This endpoint allows you to search through public add-ons.
     ==============  ==========================================================
          Parameter  Description
     ==============  ==========================================================
-           created  Creation date, descending
-         downloads  Number of weekly downloads, descending
+           created  Creation date, descending.
+         downloads  Number of weekly downloads, descending.
            hotness  Hotness (average number of users progression), descending.
             rating  Bayesian rating, descending.
-           updated  Last updated date, descending
+         relevance  Search query relevance, descending.
+           updated  Last updated date, descending.
              users  Average number of daily users, descending.
     ==============  ==========================================================
 
-    The default is to sort by number of weekly downloads, descending.
+    The default is to sort by relevance if a search query is present, otherwise
```

Comment:
Putting "search query (`q`)" so there's a mapping of which actually API param is needed would help makes this a bit clearer.